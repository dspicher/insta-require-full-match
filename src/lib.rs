#[cfg(test)]
mod tests {
    #[test]
    fn test_insta() {
        insta::assert_json_snapshot!(1+1);
    }
}
