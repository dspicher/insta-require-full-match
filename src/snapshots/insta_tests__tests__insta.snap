---
source: src/lib.rs
expression: 1 + 1
---
2
